lms/djangoapps/labster_vouchers/urls.py

```diff
@@ -0,0 +1,14 @@
+"""
+URLs for the Labster voucher system.
+"""
+from django.conf.urls import patterns, url
+
+
+urlpatterns = patterns(
+    '',
+    url(r'^enter/$',
+        'labster_vouchers.views.enter_voucher', name='enter_voucher'),
+    url(r'^activate/$',
+        'labster_vouchers.views.activate_voucher', name='activate_voucher'),
+)
+
```

Comment:
Livit.Learn.EdX/lms/djangoapps/labster_vouchers/urls.py:14:1: W391 blank line at end of file